CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+# dbt_hubspot_source v0.17.0
+
+## Breaking Change (`--full-refresh` required after upgrading)  
+- Introduced a new `category` column to the below models. This association field differentiates records by either HUBSPOT_DEFINED (default label) or USER_DEFINED (custom label) and was introduced to the Hubspot connector in October 2024. See the [connector release notes](https://fivetran.com/docs/connectors/applications/hubspot/changelog#october2024) for more.
+  - `stg_hubspot__deal_company`
+  - `stg_hubspot__deal_contact`
+  - `stg_hubspot__engagement_company`
```

Comment:
@fivetran-reneeli I do not see the `category` column added to any of the models from lines 7-13. Was the intention to add `category` to these models, or should these lines be removed?

Reply:
Great question! So these models leverage a macro to bring through all the columns and therefore the `category` field was not explicitly called out in them, thus the models themselves didn't require any changes. But you can see that the new field gets brought through:

![image](https://github.com/user-attachments/assets/e916f1af-95fd-48bd-8986-3d43d93cf54c)

Reply:
Ah makes sense! Thanks for that.